examples/convert_eeg_to_bids.py

```diff
@@ -23,10 +23,11 @@
 import os
 import shutil as sh
 
+import mne
 from mne.datasets import eegbci
-from mne.io import read_raw_edf
+from mne.io.edf.edf import read_annotations_edf
 
-from mne_bids import raw_to_bids
+from mne_bids import write_raw_bids, make_bids_basename
```

Comment:
I am not convinced that we need to expose the `make_bids_basename` to users. I think having users specify subject, run, task, and optionally session in the `write_raw_bids` is probably a better approach